plugins/kotlin/j2k/shared/tests/testData/newJ2k/lambda/lambdas.java

```diff
@@ -1,4 +1,5 @@
 // IGNORE_K2
```

Comment:
I'm guessing I'll be able to remove this directive once the postprocessing is ported to a JKTree step.